docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.8'
+
+services:
+  app:
+    build: .
+    volumes:
+      - ./benchmark:/usr/src/app
```

Comment:
**issue(blocking):** Why you're mounting this volume? Your code is already in the container (see Dockerfile):

Line 14:
```Dockerfile
COPY . .
```